recipes/earthengine-api/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "earthengine-api" %}
+{% set version = "0.1.137" %}
+{% set sha256 = "6dd2e0e8de8214c5cf499c55f13154f6a5109ef5ea673260c80b22f9a2b51c96" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
It looks like this package might be a candidate for `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
I added the noarch command.